packages/remix-dev/cli/convert-to-javascript.ts

```diff
@@ -58,12 +60,36 @@ export async function convertTemplateToJavaScript(projectDir: string) {
     }
   }
 
-  // 2. Rename the tsconfig.json to jsconfig.json
-  if (fse.existsSync(path.join(projectDir, "tsconfig.json"))) {
-    fse.renameSync(
-      path.join(projectDir, "tsconfig.json"),
-      path.join(projectDir, "jsconfig.json")
+  // 2. Rename all tsconfig.json files to jsconfig.json
+  let tsconfigs = glob.sync("**/tsconfig.json", {
+    cwd: projectDir,
+    absolute: true,
+    ignore: ["**/node_modules/**"],
+  });
+  for (let tsconfig of tsconfigs) {
+    let contents = fse.readFileSync(tsconfig, "utf8");
+    let json = JSON5.parse(contents) as TsConfigJson;
+    if (json.include) {
+      json.include = json.include
+        .map((include) => {
+          if (include === "remix.env.d.ts") return null;
+          if (include === "**/*.ts") return "**/*.js";
+          if (include === "**/*.tsx") return "**/*.jsx";
+          return include;
+        })
+        .filter((include: any): include is string => include !== null);
```

Comment:
I think this could be simplified to

```suggestion
        .filter(Boolean);
```

Reply:
typescript has troubles with just using `filter(Boolean)`